app/assets/javascripts/bootstrap_datetimepicker/dates.coffee

```diff
@@ -96,7 +96,7 @@ class DateTimePicker
     format = 'YYYY-MM-DD'
     view = 'M/D/YYYY'
     # Update Values
-    sel = $("input.date")
+    sel = $("input.date:not([data-html-5='true'])")
```

Comment:
This is opt in so making sure not to enable the js or css when html 5 is enabled